go.mod

```diff
@@ -151,10 +151,10 @@ require (
 	go.opencensus.io v0.23.0 // indirect
 	go.uber.org/atomic v1.7.0 // indirect
 	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616 // indirect
-	golang.org/x/mod v0.6.0-dev.0.20211013180041-c96bc1413d57 // indirect
-	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd // indirect
+	golang.org/x/mod v0.13.0 // indirect
+	golang.org/x/net v0.16.0 // indirect
```

Comment:
Hmm, this is 'fine' for this, but v0.17.0 has the corresponding http/2 fix for `golang.org/x/net`. Is it possible to update this PR so that `v0.17.0` is also pulled in?